.ci/atime/tests.R

```diff
@@ -135,7 +135,7 @@ test.list <- atime::atime_test_list(
     expr = {
       expr=data.table:::`[.data.table`(d, , (max(v1)-min(v2)), by = id3)
     },
-    Before="793f8545c363d222de18ac892bc7abb80154e724", # Commit (https://github.com/Rdatatable/data.table/commit/793f8545c363d222de18ac892bc7abb80154e724) which was Before the regression was introduced as stated in the issue (https://github.com/Rdatatable/data.table/issues/4200#issuecomment-646111420) 
+    Before="905e4d302be0dc326684d7228ece88064321cc00", # Parent of the regression commit (https://github.com/Rdatatable/data.table/commit/c152ced0e5799acee1589910c69c1a2c6586b95d)
```

Comment:
why did you change the Before commit ID?

The Before commit comment you wrote is "Parent of the regression commit (https://github.com/Rdatatable/data.table/commit/c152ced0e5799acee1589910c69c1a2c6586b95d)" which is not consistent with the suggested template, "Parent of first commit (COMMIT_LINK) in the PR (PR_LINK) which causes the issue" 

please re-read https://github.com/Rdatatable/data.table/wiki/Performance-testing#documentation-of-historical-commits and if there is anything that you do not understand, please ask for clarification.